addon/-private/collapse-tree.js

```diff
@@ -275,7 +277,9 @@ export const TableRowMeta = EmberObject.extend({
 function reduceSelectedRows(selection, groupingCounts, rowMetaCache) {
   let reducedGroupingCounts = new Map();
 
-  for (let [group, count] of groupingCounts.entries()) {
+  let groupingCountsEntries = Array.from(groupingCounts.entries());
```

Comment:
It is unfortunate to allocate an array for each of the maps.

But furthermore `entries` is a method on `Map`, and though IE11 has a `Map` implementation it doesn't have support for `entries`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/entries#Browser_compatibility

So there is still a need for a polyfil in IE11.